bot/bot.py

```diff
@@ -1,12 +1,38 @@
-from tokenize import Token
 import hikari
 import os
 from dotenv import load_dotenv
 
-load_dotenv()
```

Comment:
why this line was removed ?
`load_dotenv()` loads `.env` file that contains key value pairs into the the environment variables, you should use `.env` to store the bot token, and firebase secrets on your local workspace. Check [dotenv](https://github.com/theskumar/python-dotenv) module